### ISSUE TITLE
Examples

### ISSUE THREAD

**Issue by benjaminapetersen** [jgthms/bulma#1727]
One handy feature, coming from Bootstrap, is the top nav item [Examples](http://getbootstrap.com/docs/4.0/examples/)" with +/- 20 page layout templates built with Bootstrap.  I'm finding that its taking a rather significant amount of time just poking around the Bulma docs trying to find the bits I need.  Since I don't know Bulma yet, I'm not 100% certain what I'm looking for actually exists.  This is where a small handful of example page templates would be enormously helpful.  

Thanks!

**Comment by jgthms** [comment]
You're right! It's in the pipelines, but I haven't time to create proper templates. I want them to be of high quality.

Check out http://html.mijnspeelplek.com/bulma1/ in the meantime.

**Comment by benjaminapetersen** [comment]
Will do!  Yeah, even one to begin with would be super helpful.  

Thanks!

**Comment by benjaminapetersen** [comment]
Yup, I think that's what I'm looking for. Thanks again.

**Comment by benjaminapetersen** [comment]
Oh... I dunno if I would close this until there is something official on the website.  I def opened it as a feature suggestion as "Examples" would be part of official docs, rather than having to hunt around elsewhere to find things.

**Comment by benjaminapetersen** [comment]
Just noticed the link http://html.mijnspeelplek.com/bulma1/ is 404 now.
